### PR TITLE
Max

### PR DESCRIPTION
**Pull request by RyoMax**

### CODE REVIEW

File: src/pages/Dashboard/DashboardPage.tsx [hunk 1]
```diff
@@ -121,9 +121,9 @@ const DashboardPage = () : JSX.Element => {
           <AppointmentArea />
         </Item>
         <Item sx={{ gridArea: 'ui' }}>
-          <Box sx={{ m: 2 }}>
-            User information / Number of appointments left.
-          </Box>
+
```
Comment (tkalejandro) [inline src/pages/Dashboard/DashboardPage.tsx]: remove empty space please :-)

File: src/pages/Dashboard/DashboardPage.tsx [hunk 2]
```diff
@@ -121,9 +121,9 @@ const DashboardPage = () : JSX.Element => {
           <AppointmentArea />
         </Item>
         <Item sx={{ gridArea: 'ui' }}>
-          <Box sx={{ m: 2 }}>
-            User information / Number of appointments left.
-          </Box>
+ 
+          <UserInformation />
+
```
Comment (tkalejandro) [inline src/pages/Dashboard/DashboardPage.tsx]: remove empty space

File: src/pages/Dashboard/components/UserInformation/UserInformation.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import { Box, useTheme, /* styled, Paper, */ Grid, useMediaQuery, Typography } from '@mui/material'
+import AccountCircleIcon from '@mui/icons-material/AccountCircle';
+import React from 'react'
+
+
+
```
Comment (tkalejandro) [inline src/pages/Dashboard/components/UserInformation/UserInformation.tsx:7]: remove empty space - 1. Always. 2 spaces between imports and the component

File: src/pages/Dashboard/components/UserInformation/UserInformation.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    /* const Item = styled(Paper)({
+        padding: theme.spacing(1),
+        textAlign: 'center',
+        color: theme.palette.text.secondary,
+    }); */
+
+    const isMd = useMediaQuery(theme.breakpoints.up('md'));
+    return (
+        <Box sx={{ m: 2 }}>
+            User information / Number of appointments left.
```
Comment (tkalejandro) [inline src/pages/Dashboard/components/UserInformation/UserInformation.tsx]: Can you initiate translations???  you need to use the:

const {t} = useTranslation()

please check how I did it in the other components. This is very important to learn!

For spanish:  Informacion de usuario

File: src/pages/Dashboard/components/UserInformation/UserInformation.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                }}>
+                    <AccountCircleIcon sx={{ fontSize: isMd ? 250 : 200, color: 'primary' }} />
+                    <Typography mt={2} variant={isMd ? 'h3' : 'h4'}>
+                        Trevor Phillips
+                    </Typography>
+                </Box>
+                <Typography mt={4}>
+                    {/* available currently hardcoded */}
+                    <Typography color='primary' variant='h5' component='span'>3 </Typography>
+                    appointments available
```
Comment (tkalejandro) [inline src/pages/Dashboard/components/UserInformation/UserInformation.tsx]: Please use translation. probably switch the order for the moment. So the number is in the right side.

for spanish:  "Citas restante:"

File: src/pages/Dashboard/components/UserInformation/UserInformation.tsx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+                </Typography>
+            </Box>
+            <Grid container spacing={2} mt={4}>
+                {/* User Profile Picture and name */}
+                <Grid item xs={6} lg={6}>
+                    <Typography textAlign="center">
+                        Age
```
Comment (tkalejandro) [inline src/pages/Dashboard/components/UserInformation/UserInformation.tsx]: use translation.

Spanish:  Edad

**Comment by tkalejandro** [comment]
test

**Comment by RyoMax** [comment]
Integrated all suggestion - should be fine now

**Review by tkalejandro — APPROVED** [review]
Everything good! 

Make sure the next branch name is not "max" but "the thing you are doing".

**Pull request merged**